src/CarbonAware.CLI/src/CarbonAware.CLI.csproj

```diff
@@ -1,37 +1,50 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <AssemblyName>caw</AssemblyName>
+    <RuntimeIdentifiers>win-x64;osx-x64;linux-x64</RuntimeIdentifiers>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
-    <AssemblyName>CarbonAware.CLI</AssemblyName>
     <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <UserSecretsId>34d82203-20b1-4fcd-9bd4-3b247f13bad7</UserSecretsId>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Remove="build\**" />
-    <EmbeddedResource Remove="build\**" />
-    <None Remove="build\**" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
```

Comment:
nit: Seems like we are using Extensions version `6.0.0` on other projects. It would be good to keep it like others or move the rest to a new one (this one)